src/ble/CHIPBleServiceData.h

```diff
@@ -45,11 +45,12 @@ enum chipBLEServiceDataType
  */
 struct ChipBLEDeviceIdentificationInfo
 {
-    constexpr static uint16_t kDiscriminatorMask     = 0xfff;
-    constexpr static uint8_t kAdditionalDataFlagMask = 0x1;
+    constexpr static uint16_t kDiscriminatorMask        = 0xfff;
+    constexpr static uint8_t kAdditionalDataFlagMask    = 0x1;
+    constexpr static uint16_t kAdvertisementVersionMask = 0xf000;
 
     uint8_t OpCode;
-    uint8_t DeviceDiscriminator[2];
+    uint8_t DeviceDiscriminator[2]; // 12 bits for device discriminator and 4 bits for advertisement version
```

Comment:
This member used to be renamed if it's not just the discriminator anymore.

Reply:
sure